pkg/provider/cloud/vsphere/credentials.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
```

Comment:
```suggestion
Copyright 2023 The Kubermatic Kubernetes Platform contributors.
```

This is usually everyone's lease favorite nit pick :D